test/integration/targets/bigip_gtm_wide_ip/tasks/environment/issue-persistence.yaml

```diff
@@ -0,0 +1,124 @@
+---
+
+- name: Issue Persistence - Include issue variables
+  include_vars:
+    file: issue-persistence.yaml
+
+- name: Issue Persistence - Provision GTM
+  bigip_provision:
+    module: gtm
+  tags:
+    - module-provisioning
+
+- name: Issue Persistence - Collect BIG-IP facts
+  bigip_device_facts:
+    include: system-info
+  register: f
+
+- name: Issue Persistence - Create pool, type 'a'
+  bigip_gtm_pool:
+    preferred_lb_method: "{{ pool_lb_method }}"
+    name: "{{ pool_name }}"
+    state: present
+    type: a
+  when: f.system_info.product_version >= "12.0.0"
+  register: result
+
+- name: Issue Persistence - Create wide IP with persistence
+  bigip_gtm_wide_ip:
+    lb_method: "{{ pool_lb_method }}"
+    state: present
+    type: a
+    name: "{{ wide_ip1 }}"
+    persistence: True
+    persist_cidr_ipv4: 24
+    persist_cidr_ipv6: 120
+    pools:
+      - name: "{{ pool_name }}"
+  when: f.system_info.product_version  >= "12.0.0"
+  register: result
+
+- name: Issue Persistence - Assert Create wide IP with persistence
+  assert:
+    that:
+      - result is changed
+  when: f.system_info.product_version  >= "12.0.0"
+
+- name: Issue Persistence - Create wide IP with persistence - Idempotent check
+  bigip_gtm_wide_ip:
+    lb_method: "{{ pool_lb_method }}"
+    state: present
+    type: a
+    name: "{{ wide_ip1 }}"
+    persistence: True
+    persist_cidr_ipv4: 24
+    persist_cidr_ipv6: 120
+    pools:
+      - name: "{{ pool_name }}"
+  when: f.system_info.product_version  >= "12.0.0"
+  register: result
+
+- name: Issue Persistence - Assert Create wide IP with persistence - Idempotent check
+  assert:
+    that:
+      - result is not changed
+  when: f.system_info.product_version  >= "12.0.0"
+
+
+- name: Issue Persistence - Create wide IP without persistence
+  bigip_gtm_wide_ip:
+    lb_method: "{{ pool_lb_method }}"
+    state: present
+    type: a
+    name: "{{ wide_ip2 }}"
+    pools:
+      - name: "{{ pool_name }}"
+  when: f.system_info.product_version  >= "12.0.0"
```

Comment:
this kind of check needs to be done before the test is run

Reply:
I'm just following one of the recent standards here - https://github.com/F5Networks/f5-ansible/blob/ba8e3002abe6d3075a611c7dc0427bb77ccb2281/test/integration/targets/bigip_network_globals/tasks/main.yaml

Reply:
this would make sense if you had a supported version in mind and some tasks were supposed to run on both versions, since you want all to run on v12 and beyond then it is not needed, also  v11 is deprecated so we can get rid of this :)